i18n/fr.yaml

```diff
@@ -91,11 +91,11 @@ rejoindre-notre-equipe:
 ce-que-nous-faisons:
   other: "Ce que nous faisons"
 contact-us:
-  other: "Rejoindre la conversation"
+  other: "Participez à la conversation"
 featured-post:
-  other: "Articles de blog récents"
+  other: "Récents billets de blogue"
 more-posts:
-  other: "Voir tout"
+  other: "Voir tous les produits"
```

Comment:
It seems that it's the same button for Blog and Products. But for blog posts it should say "Voir tous les billets" and for products it should say "Voir tous les produits"